app/web/components/Footer/Footer.tsx

```diff
@@ -177,6 +178,15 @@ export default function Footer() {
               variant="outlined"
               color="primary"
               className={classes.button}
+              sx={{
+                color: theme.palette.common.black,
+                borderColor: theme.palette.grey[300],
+
+                "&:hover": {
+                  borderColor: theme.palette.grey[300],
+                  backgroundColor: "#3135390A",
+                },
+              }}
```

Comment:
Another big change is MUI made our theme primary color the default for buttons. This turned our black/gray buttons green since our primary color is green. Our secondary is orange. So you'll see me adding these custom styles with the new spiffy `sx` prop.